language/Portuguese_Brazil/strings.po

```diff
@@ -516,27 +516,27 @@ msgstr "Opções do State/Playback"
 
 #: src/frontend/mame/ui/submenu.cpp:76
 msgid "Automatic save/restore"
-msgstr "Salve/restaure automático"
+msgstr "Salvamento/Restauração Automáticos"
 
 #: src/frontend/mame/ui/submenu.cpp:77
 msgid "Allow rewind"
-msgstr "Permita o rebobinamento"
+msgstr "Permitir o retrocesso"
```

Comment:
"rebobinamento" was better, I think.

But maybe it could be "Permitir rebobinar"